test/fixtures/native-overrides-test/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "native-overrides-test",
+  "main": "index.js",
+  "dependencies": {
+    "foo": "*",
+    "fs-extra": "*"
+  },
+  "jetpack": {
+    "overrides": {
+      "fs": "sdk/io/fs",
+      "overload": "foo"
```

Comment:
Does this cover the scenario where we have `"sdk/tabs": "./tabs"`, where you're using your own version of "fs" or possibly an sdk built-in, or just want global access to a module? I think that was the complexity part I ran into before, but if this supports that scenario, then we should test it, otherwise, have a separate bug for it